### ISSUE TITLE
Usernames being styled in the same way as the post content could be ambiguous

### ISSUE THREAD

**Issue by JamieCrisman** [cblgh/cerca#7]
So the image is of a single post with the content:

```
blep

**ciel**

ok... spacing might not be the same
```

<img width="528" alt="スクリーンショット 2022-01-12 8 36 33" src="https://user-images.githubusercontent.com/697955/149038105-bfe549b6-0a2d-424b-a564-e7c7d322666d.png">

I don't believe our members will be malicious with this, but current styling makes the usernames and the content ambiguous. 
Of course, spacing isn't the exact same of when a user actually posts. But it might be worth it to make it visually obvious to prevent (even accidental) confusion.

**Comment by Thomasorus** [comment]
Normally the date should also be visible in a future update and should help with this. Obviously it's a design issue among several others on readability, we will have to look into it.

**Comment by sansfontieres** [comment]
Having a divider (like a rule or borders) rather than pure spacing would improve the situation.

Maybe using a flexbox to put the metadata (username, date) on the right of posts could also help (ping @ftrvxmtrx @neauoire, they work on mothra and netsurf iirc)

**Comment by Thomasorus** [comment]
> Maybe using a flexbox to put the metadata (username, date) on the right of posts could also help (ping @ftrvxmtrx @neauoire, they work on mothra and netsurf iirc)

I'm willing to do a rewrite of the whole frontend to propose graceful degradation for those browsers. Not a lot of work once we have a few utility classes for layouts. :3

**Comment by sansfontieres** [comment]
I suppose that you would have to do the reverse of the `@support` css class? I guess that overall pages would still be light considering that the template is small to begin with :)

**Comment by Thomasorus** [comment]
Yes, you reverse support for old browsers. You write in floats first, then use support for recent browsers to turn the class into flex or grid. It won't be huge work, the template is rather simple.

**Comment by neauoire** [comment]
I'm guessing that usernames will always be prefixed with a rune like `@neauoire`

**Comment by ftrvxmtrx** [comment]
I have no idea about any frontend terms discussed here, so can't really say what's going to work (or not) with mothra/netsurf on plan 9 :) sorry

**Comment by Thomasorus** [comment]
Just so you know: I've changed the usernames to have an "@" before them and proposed a pull request to @cblgh. He wasn't super hot on the idea of using this for the forum and (rightfully) removed it from my PR. In the meantime the date with a link on it was added, helping a little bit, but it turns out it's quite easy to impersonate another user:

![Screenshot 2022-01-14 at 12-30-34 Forum — The Test Thread figure things out to your heart's desire](https://user-images.githubusercontent.com/8302525/149508768-d9de7f03-1e9e-48f1-9c3c-948e4f62d411.png)

So, in order to resolve this issue, we should try to find another way of identifying users in a way that can't be reproduced using markdown in the forum.

Any ideas?

**Comment by sansfontieres** [comment]
It’s more a question of identifying posts (dividing them visually) than users imho. A naive approach would be to use `:not(article) + hr` to separate posts, and/or a metadata column on the right of posts (you would still need a vertical visual separator for mobile)

<img width="817" alt="Screenshot 2022-01-14 at 13 03 05" src="https://user-images.githubusercontent.com/83609756/149512740-d4667caf-b1ba-4cbe-ac46-946f6301ed40.png">

**Comment by ftrvxmtrx** [comment]
How about making the timestamp something other than `today` and align it right? Would that be any better indication? I'd like to believe we don't have to be defensive with the idea that someone in Merveilles will start impersonating others.

**Comment by sansfontieres** [comment]
Right, I think we are pretty safe regarding impersonation. Regarding mentions, though, I think the issue will be quasi-fixed once wiki features will come (I think it was discussed?).

**Comment by cblgh** [comment]
I quite like @sansfontieres [redesign](https://github.com/cblgh/cerca/issues/7#issuecomment-1013061998) that sets the names at the left 😊 

As others have said, I also agree that impersonation is not a high priority concern for a close(d/-knit) community.

> It’s more a question of identifying posts (dividing them visually) than users imho

Agree, though I would really like to maintain the cohesiveness of posts the forum has at the moment—I think the lack of borders etc has a more collaborative vibe than each person living in their own little box, espousing their own little opinions. It's more of a conversation, than shouting into the void.

---

If people are seriously confused while reading posts, not knowing where one ends and the other starts, even some small visual indicator like

![image](https://user-images.githubusercontent.com/3862362/149515456-4e59fc04-3f14-4e57-90e0-dd15fbc369c0.png)

could suffice

**Comment by Thomasorus** [comment]
@cblgh Want me to do a little PR for this?
